.travis.yml

```diff
@@ -125,7 +125,7 @@ jobs:
     - stage: test
       env: NAME="esp32 port build"
       install:
-        - sudo pip install pyparsing
```

Comment:
There is a requirements.txt in ESP-IDF if that matters.

Reply:
thanks, I'm trying to optimise the total time, so only installing those things that are necessary